helm/ipt-s3/templates/deployment.yaml

```diff
@@ -50,14 +50,14 @@ spec:
                 name: s3-secret
                 key: s3-secret-key
           - name: S3_ZIP_BUCKET_NAME
-            value: {{ .Values.zipBucket}}
+            value: {{ .Values.zipBucket }}
         resources:
           limits:
-            memory: 256Mi
-            cpu: "0.2"
+            memory: {{ .Values.sidecarLimitsMemory }}
```

Comment:
@MichalTorma do we still need this do you think?

Reply:
I hope not... let's default to not having it and if we find some more issues (I know, highly unlikely). let's hope `mc` is not that hungry...